governator-core/src/test/java/com/netflix/governator/package1/AutoBindSingletonMultiBinding.java

```diff
@@ -0,0 +1,11 @@
+package com.netflix.governator.package1;
+
+import com.netflix.governator.annotations.AutoBindSingleton;
+
+import javax.inject.Singleton;
+
+@AutoBindSingleton(multiple=true, value=AutoBindSingletonInterface.class)
+@Singleton
+public class AutoBindSingletonMultiBinding implements AutoBindSingletonInterface {
```

Comment:
@AutoBindSingleton isn't actually a scope.  So, using this class without auto binding will not have singleton behavior.  Unfortunately we can't make AutoBindSingleton a scope since there's already code out there that has both and having two scopes on a class is not allowed in Guice.